### PR TITLE
Fix variable names in various files

### PR DESCRIPTION
**Pull request by kulcsaradam**
Signed-off-by: Adam Laszlo Kulcsar <kuladam@inf.u-szeged.hu>

### CODE REVIEW

File: src/jit/Analysis.cpp [hunk 1]
```diff
@@ -579,61 +579,61 @@ void JITCompiler::computeOperandLocations(JITFunction* jit_func,
...
-                    stack_allocator->push(operand->location.valueInfo);
-                    operand->value = stack_allocator->values().back().value;
+                    m_stackAllocator->push(operand->location.valueInfo);
+                    operand->value = m_stackAllocator->values().back().value;
                     break;
                 }
                 case Operand::Register: {
-                    stack_allocator->pushReg(operand->value,
+                    m_stackAllocator->pushReg(operand->value,
                                              operand->location.valueInfo);
```
Comment (zherczeg) [inline src/jit/Analysis.cpp:588]: Something is wrong with alignment, did you run the style checker?

Comment (kulcsaradam) [reply src/jit/Analysis.cpp:588]: No, I forgot that I did not have it installed on the machine I am working on. I am fixing this right now.

**Comment by kulcsaradam** [comment]
The style errors should be fixed now.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**